src/channel.js

```diff
@@ -0,0 +1 @@
+module.exports = '__THEMING__';
```

Comment:
@iamstarkov I think it's a separate discussion altogether, but I'm quite fond of the idea of abstracting the context key, so that conflicts are highly unlikely